### DISCUSSION TITLE
Hyprland with additions

### DISCUSSION THREAD

**Discussion by dieBakterie** [kRHYME7/Hyde-cli#42] [Q&A]
If you've an different structure than HyDE with additional files in the hypr folder and additional folders `~/.config/hyprlock` for example do i need to maintain a fork then or can i use just add those in the Clone directory and the `manage_cfg.lst`, `restore_cfg.lst`?

**Comment by kRHYME7** [comment]
>manage_cfg.lst

This is Dynamic and the interactive CLI will write changes to manage_cfg.lst do not touch this file. 

> restore_cfg.lst

This one is use to manage the dirs and files on the Clone repo and used by Hyde so you can add any file you want here.  the CLI will detect any changes for you.

**Reply by dieBakterie** [reply]
So a fork with those added are the easiest way to go, or can i just add those to the cloned dir without getting them removed when updating(`git pull`)?

**Reply by kRHYME7** [reply]
Why will it be removed during git pull ?

BTW why are you hustling about other custom config without planning to actually fork Hyde repo? I mean, why do you bother to add some userprefs to  the ./restore_cfg.lst? FYI, Hyde will never touch files that are not declared inside ` ./restore_cfg.lst `. So if you have custom files and are not declared inside ./restore_cfg.lst don't add it. unless you know what you're doing. 

Or I'm getting the question wrong,  are you asking about how ` git ` works?

**Reply by dieBakterie** [reply]
Yeah don't know how Git(hub) works and for me a fork is more work then another repo only with my changes and  additions.

**Reply by kRHYME7** [reply]
For completeness, https://github.com/kRHYME7/Hyde-cli/issues/41#issuecomment-2081942238
